### ISSUE TITLE
CVE's in current docker image

### ISSUE THREAD

**Issue by ndoornekamp** [amacneil/dbmate#598]
Hi there, our CVE scanner detects a CVE in the latest docker image:
 https://nvd.nist.gov/vuln/detail/CVE-2024-10979, which should be fixed if we upgrade package `postgresql16-client` to version 16.5-r0 or above and `postgresql-client-15` to version `15.9-0+deb12u1` or above.

Could a patch be released with upgraded versions?

**Comment by dossy** [comment]
Good news: there's no urgency to upgrade client applications like `dbmate` in response to CVE-2024-10979.

It isn't a client-side vulnerability, but a server-side arbitrary file execution vulnerability and only if PL/Python functions are enabled accessible to the user connecting to the server.

Thanks for raising the issue and making us aware of it, though!  Very helpful, especially for anyone who is responsible for running a Postgres server that has PL/Python enabled and users permitted to use it.

**Comment by ndoornekamp** [comment]
This was resolved in https://github.com/amacneil/dbmate/releases/tag/v2.24.0

**Comment by amacneil** [comment]
Yeah its quite annoying that vulnerability scanners report any CVE in the golang standard library or our dependencies whether or not they affect dbmate. They err on the side of caution to the point of being useless.

That said, I generally do a monthly release to keep up to date with these since our own vulnerability scanner also complains.

**Comment by ndoornekamp** [comment]
That's a good way of putting it and a suitable way of dealing with it imo. Thanks @amacneil and @dossy
